release/models/bgp/openconfig-bgp-common.yang

```diff
@@ -260,8 +275,19 @@ submodule openconfig-bgp-common {
     }
 
     leaf send-community {
+      status deprecated;
+      type oc-bgp-types:community-type;
+      description
+        "This leaf has been deprecated and replaced by send-community-type to
+        support large communities.
+
+        Specify which types of community should be sent to the
+        neighbor or group. The default is to not send the
+        community attribute";
+    }
+
+    leaf-list send-community-type {
       type oc-bgp-types:community-type;
-      default "NONE";
```

Comment:
should set default to NONE for send-community-type

Reply:
The empty leaf-list already means no community type should be sent, I do not think we need a default value here. The only reason we keep the value NONE in the enum is to keep compatibility.

Reply:
Ok, that is fair.  Empty list means none.

Reply:
However, the default is removed from `leaf  send-community`.  (I know the github diff makes this look strange, but look at the file[ openconfig-bgp-common.yang in the master branch](
https://github.com/openconfig/public/blob/d1f77330648a3f6a8daac1cb6f16302007d5da92/release/models/bgp/openconfig-bgp-common.yang#L262-L269).  I think to preserve this as a non-breaking change the default for `leaf send-community` should be retained.